deployment/ansible/group_vars/all

```diff
@@ -4,6 +4,8 @@ accumulo_follower: not accumulo_leader
 developing: "'development' in group_names"
 packing: "'packer' in group_names"
 
+accumulo_mirror: "http://download.nextag.com/apache"
+
 java_version: "7u51-2.4.*"
 spark_version: "1.2.0+cdh5.3.*"
```

Comment:
Ugh, yeah -- going to update things here with the newest roles now that these other roles have been updated.